tests/common/util/http_helpers_test.py

```diff
@@ -53,16 +55,36 @@ def test_without_suffix(self):
         response, content = http.request(DUMMY_URL)
         self.assertIn(UA_KEY, mock_http.headers)
         self.assertRegex(
-            mock_http.headers[UA_KEY], r'forseti-security/[0-9.]+$')
+            mock_http.headers[UA_KEY], r'forseti-security/[0-9.]+\s+gce$')
 
-    def test_with_suffix(self):
+    def test_with_suffix_set_platform(self):
```

Comment:
Can you rename this method back to `test_with_suffix`? As it stands there are 2 methods with the same name (see L79).

Reply:
d'oh